docs/docs/configuring-usage-with-plugin-options.md

```diff
@@ -54,6 +54,24 @@ The following table lists possible options values and an example plugin that mak
 
 **Note**: Themes (which are a type of plugin) are able to receive options from a site's `gatsby-config` to be used in its `gatsby-config` in order to allow themes to be composed together. This is done by exporting the `gatsby-config` as a function instead of an object. You can see an example of this in the [`gatsby-theme-blog`](https://github.com/gatsbyjs/themes/tree/master/packages/gatsby-theme-blog) and [`gatsby-theme-blog-core`](https://github.com/gatsbyjs/themes/tree/master/packages/gatsby-theme-blog-core) repositories. Plugins are not capable of this functionality.
 
+## How to validate options
+
+A Gatsby plugin can define a [Joi](https://joi.dev) schema for their options using the `pluginOptionsSchema` API in `gatsby-node.js`. Gatsby validates that the options users of the plugin pass to it match the schema.
```

Comment:
Also is there a separate doc for the `pluginOptionsSchema` API that we can link to here?

Reply:
Not yet! That's going to be the next step after this 👍